src/main/java/no/nav/pto/veilarbportefolje/persononinfo/PdlService.java

```diff
@@ -52,18 +50,12 @@ public void hentOgLagreBrukerDataPaBarn(Fnr fnrBarn) {
 
     public void lagreBrukerData(Fnr fnrPerson, PDLPerson personData) {
         pdlPersonRepository.upsertPerson(fnrPerson, personData);
-
-        List<BarnUnder18Aar> barn = new ArrayList<>();
```

Comment:
Why is all this removed?

Reply:
I thought to fetch data from PDL inside barnUnder18AarService when its needed.